libcodechecker/session_manager.py

```diff
@@ -57,18 +57,22 @@ def calc_persistency_hash(auth_string):
         return hashlib.sha256(auth_string + "@" +
                               _Session.__initial_salt).hexdigest()
 
-    def __init__(self, token, phash):
+    def __init__(self, token, phash, user, authenticated):
+        self.authenticated = authenticated
+        self.user = user
         self.token = token
         self.persistent_hash = phash
         self.last_access = datetime.now()
+        self.destroyed = False
```

Comment:
We were trying with explicit logouts. Could have been a left-behind. We weren't sure with @dkrupp if the logout feature not working is a result of multithreading errors and sync issues or something else.

The answer was simple, mainly that `POST` queries don't send a new cookie, only `GET`, and that `HTTP-Authenticate` wasn't planned to support a "Logout" anyways. (The browser **always** sends the username-password header in **every** request once it caches it, so even if a session serverside is invalidated explicitly, the browser always tries to reauthorize.)